src/main/java/com/reverse/userservice/controllers/UserController.java

```diff
@@ -89,7 +87,7 @@ public ResponseEntity editUser(@RequestBody UserEdit userEdit) {
         }
     }
 
-    @PostMapping(path = "/getUser", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
+    @GetMapping(path = "/getUser", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Make sure you are checking for JWTs in the request on places that need it! UserService has some classes written for that